src/main/java/com/linkedin/kmf/KafkaMonitor.java

```diff
@@ -79,8 +90,11 @@ public KafkaMonitor(Map<String, Map> testProps) throws Exception {
     List<MetricsReporter> reporters = new ArrayList<>();
     reporters.add(new JmxReporter(JMX_PREFIX));
     Metrics metrics = new Metrics(new MetricConfig(), reporters, new SystemTime());
-    metrics.addMetric(metrics.metricName("offline-runnable-count", METRIC_GROUP_NAME, "The number of Service/App that are not fully running"),
-      (config, now) -> _offlineRunnables.size());
+    Map<String, String> tags = new HashMap<>();
```

Comment:
Nit: If there is only one value in the map, you can use `Map<String, String> tags = new HashMap<>(1)`

Reply:
Updated.